src/migrations/20181211163828-create-tags.js

```diff
@@ -0,0 +1,22 @@
+export default {
+  up: (queryInterface, Sequelize) => queryInterface.createTable('tags', {
+    id: {
+      allowNull: false,
+      primaryKey: true,
+      type: Sequelize.UUID
+    },
+    tagName: {
```

Comment:
I think it will be nice to have the length of the tag checked so it doesn't exceed 20 characters.

Reply:
okay, will implement that